shared/js/multi_sim_action_button.js

```diff
@@ -117,11 +117,9 @@ MultiSimActionButton.prototype._updateUI = function() {
       var self = this;
       var l10nId = this._simIndication.dataset.l10nId ||
                    'gaia-sim-picker-button';
-      navigator.mozL10n.ready(function() {
```

Comment:
be careful as this is used in Dialer (the "call" button) and Contacts (the "dial" button in a contact page) too... So maybe we'll want to keep it